test-clients/src/main/java/com/hedera/services/bdd/suites/contract/SmartContractInlineAssemblySpec.java

```diff
@@ -20,135 +20,271 @@
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.mintToken;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.scheduleCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.scheduleSign;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenAssociate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenCreate;
+import static com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoTransfer.tinyBarsFromTo;
+import static com.hedera.services.bdd.spec.transactions.token.TokenMovement.moving;
+import static com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.newKeyNamed;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.overriding;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.NOT_SUPPORTED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_TRANSFER_LIST_SIZE_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSFER_LIST_SIZE_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNRESOLVABLE_REQUIRED_SIGNERS;
 
 public class SmartContractInlineAssemblySpec extends HapiApiSuite {
 	private static final Logger log = LogManager.getLogger(SmartContractFailFirstSpec.class);
 
-	public static void main(String... args) {
-		new org.ethereum.crypto.HashUtil();
+	private static final int TMP_MAX_TRANSFER_LENGTH = 2;
+	private static final int TMP_MAX_TOKEN_TRANSFER_LENGTH = 2;
+
+	private static final String defaultTxExpiry =
+			HapiSpecSetup.getDefaultNodeProps().get("ledger.schedule.txExpiryTimeSecs");
+	private static final String defaultMaxTransferLen =
+			HapiSpecSetup.getDefaultNodeProps().get("ledger.transfers.maxLen");
+	private static final String defaultMaxTokenTransferLen =
+			HapiSpecSetup.getDefaultNodeProps().get("ledger.tokenTransfers.maxLen");
+	private static final String defaultWhitelist =
+			HapiSpecSetup.getDefaultNodeProps().get("scheduling.whitelist");
+
+	private static final String A_TOKEN = "token";
+	String failingTxn = "failingTxn";
 
+	public static void main(String... args) {
 		new SmartContractInlineAssemblySpec().runSuiteSync();
 	}
 
 	@Override
 	protected List<HapiApiSpec> getSpecsInSuite() {
 		return List.of(new HapiApiSpec[] {
-				smartContractInlineAssemblyCheck()
+				/* Stateful specs from ScheduleExecutionSpecs */
+				scheduledUniqueMintFailsWithNftsDisabled(),
+				scheduledUniqueBurnFailsWithNftsDisabled(),
+				scheduledBurnWithInvalidTokenThrowsUnresolvableSigners(),
+				executionWithTransferListWrongSizedFails(),
+				executionWithTokenTransferListSizeExceedFails(),
+				suiteCleanup()
```

Comment:
I might be missing something. Seems to me this suite's test specs are completely replaced with token operations. Maybe we need to name the class name as the original test spec name `SmartContractInlineAssemblySpec` is misleading now.